e2e/schematics/command-line.test.ts

```diff
@@ -248,4 +252,57 @@ describe('Command line', () => {
     },
     1000000
   );
+
+  describe('visualization', () => {
```

Comment:
the names seem inconsistent: visualization or dep-graph. I'd use "dip-graph" everywhere.